src/text-area/text-area.stories.mdx

```diff
@@ -182,20 +183,29 @@ Note that these variables are shared with other components such as `Textfield`,
 </Canvas>
 
 export function AutoExpandStory(props) {
+    const [value, setValue] = React.useState('')
     return (
-        <Stack space="xxlarge" dividers="secondary" maxWidth="medium">
+        <Stack space="large" dividers="secondary" maxWidth="medium">
             <TextArea
-                {...props}
-                label="What do you want to accomplish?"
-                message="Write as much or as little as you want. The input area will auto-expand to fit what you've typed."
+                label="Text area with auto-expand"
+                auxiliaryLabel="(controlled)"
                 autoExpand
+                value={value}
+                onChange={(event) => setValue(event.target.value)}
+                message="Write as much or as little as you want. The input area will auto-expand to fit what you've typed."
+                rows={1}
+                onKeyDown={(event) => {
+                    if (event.key === 'Enter') {
+                        event.preventDefault()
+                        setValue('') // Clear the input programmatically
+                    }
+                }}
```

Comment:
💭 I've noticed that this textarea doesn't have a max-width anymore in storybook, in the sense that it extends further than before. Not sure if it means anything 😅

<table>
<thead><tr><th>Before</th><th>After</th></tr></thead>
<tbody>
<tr><td>

![Screenshot 2025-04-30 at 11 18 47](https://github.com/user-attachments/assets/ddda9619-3a36-4be5-857f-5f024d8649ff)


</td><td>


![Screenshot 2025-04-30 at 11 18 38](https://github.com/user-attachments/assets/bc67ac7d-8922-4be3-aa32-3e45bf6f0c8a)




</td></tr>
</table>

Reply:
Mmm it seems to be because of the different `max-width` value compared to before (prod):

<table>
<thead><tr><th>Before</th><th>After</th></tr></thead>
<tbody>
<tr><td>

![Screenshot 2025-04-30 at 11 23 52](https://github.com/user-attachments/assets/80cd7262-1be8-4f45-a1da-65a21847aed4)

</td><td>


![Screenshot 2025-04-30 at 11 23 17](https://github.com/user-attachments/assets/12beaf2c-2f65-430c-bb29-a5243cc7521c)

</td></tr>
</table>

Reply:
Oh I guess it's because we're not passing `{...props}` anymore to the `textarea` in this story. Was that intentional?

Reply:
My mistake! Must have removed the `{...props}` inadvertently.